fairlearn/datasets/_fetch_boston.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) Microsoft Corporation and contributors. Vincent D. Warmerdam.
```

Comment:
We have just finished another round with lawyers, so we have a bit more clarity here, but we haven't yet started an issue / PR on that, so you can leave as is. But you can also consider one of the following two options for the copyright line:
```
# Copyright (c) Microsoft Corporation and Fairlearn Contributors.
# Copyright (c) Microsoft Corporation, Fairlearn Contributors, and Vincent D. Warmerdam.
```

Reply:
Wasn't Fairlearn Contributors meant to capture everyone?

Reply:
That's definitely the idea--the hope is that "Fairlearn contributors" is inclusive enough (I edited the capitalization pattern to "Fairlearn contributors").

However, I also don't see a problem explicitly listing anybody who'd like to be explicitly listed.

Reply:
To me it doesn't make sense to make a distinction between "fairlearn contributors" and "Microsoft corporation" in that case. Either "fairlearn contributors" is inclusive enough to capture everyone, including Microsoft, or it isn't.

Reply:
I'll keep it at 

```
# Copyright (c) Microsoft Corporation and Fairlearn contributors.
```

for now.